src/pages/Defi/hooks/useEarnBalances.tsx

```diff
@@ -28,17 +28,24 @@ export function useEarnBalances(): UseEarnBalancesReturn {
     useCosmosStakingBalances({
       assetId: 'cosmos:cosmoshub-4/slip44:118',
     })
+  const {
+    cosmosStakingOpportunities: osmosisStakingOpportunities,
+    totalBalance: totalOsmosisStakingBalance,
+  } = useCosmosStakingBalances({
+    assetId: 'cosmos:osmosis-1/slip44:118',
+  })
 
   // cosmosStakingOpportunities intentionally set to empty array => we do not need to display staking opportunities with no staking amount
   const opportunities = useNormalizeOpportunities({
     vaultArray,
     foxyArray,
-    cosmosStakingOpportunities,
+    cosmosStakingOpportunities: cosmosStakingOpportunities.concat(osmosisStakingOpportunities),
```

Comment:
We can now use the `cosmosAssetId` and `osmosisAssetId` constants from `shapeshiftoss/caip` instead of stings in the two instances in this file.